src/dtemplate.d

```diff
@@ -1774,11 +1774,30 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
                                 Expression e;
                                 Type t;
                                 Dsymbol s;
-                                taa.index.resolve(instLoc, sc, &e, &t, &s);
+                                Scope *sco;
+
+                                uint errors = global.startGagging();
+                                /* Bugzilla 11118:
```

Comment:
Please use link instead:
```
https://issues.dlang.org/show_bug.cgi?id=11118
```
because then examining the report is a one-click process.